qiling/os/memory.py

```diff
@@ -281,12 +281,7 @@ def write(self, addr: int, data: bytes) -> None:
             data: bytes to write
         """
 
-        try:
-            self.ql.uc.mem_write(addr, data)
-        except:
-            self.ql.log.debug(f'addresss write length: {len(data):d}')
-            self.ql.log.error(f'addresss write error: {addr:#x}')
```

Comment:
Better for us to keep the debug msg ?

Reply:
It's a matter of approach. I would suggest to leave it unguarded and let the user decide whether to catch exceptions or let their program crash.
Anyway, `read` and `write` should share the same approach for consistency.